geometry_msgs/PoseStamped goal
---
bool status